flux/src/inventory/index.tsx

```diff
@@ -99,7 +99,8 @@ export function GetResourcesFromInventory(props: {
           },
           {
             header: 'Age',
-            accessorFn: item => <DateLabel date={item?.metadata?.creationTimestamp} />,
+            accessorKey: 'metadata.creationTimestamp',
+            Cell: ({ cell }: any) => <DateLabel date={cell.getValue()} />,
```

Comment:
<img width="225" alt="Screenshot 2025-02-20 at 12 00 25 PM" src="https://github.com/user-attachments/assets/a291ab1e-28d3-4c8d-9847-7985d8d51e2d" />
The inventory age column still doesn't seem to be sorted